doc/man3/PKCS7_verify.pod

```diff
@@ -17,8 +17,8 @@ PKCS7_verify, PKCS7_get0_signers - verify a PKCS#7 signedData structure
 
 PKCS7_verify() is very similar to L<CMS_verify(3)>.
 It verifies a PKCS#7 signedData structure given in I<p7>.
-The optional I<certs> parameter refers to a set of certificates
-in which to search for signer's certificates.
+The optional I<certs> parameter can provide a list of certificates,
+which is searched first for the signer's certificate.
```

Comment:
```suggestion
which is searched first for the signer certificates.
```

Reply:
As above, changed to `which is searched first for signer certificates.`